installation/windows.md

```diff
@@ -37,7 +37,7 @@ On this page, we will be using `C:\openHAB2` as an example.
 The openHAB runtime is distributed using a platform-independent zip file.
 To install it, follow these simple steps:
 
-1.  Choose between the Stable Version [![Download](https://api.bintray.com/packages/openhab/mvn/openhab-distro/images/download.svg)](https://bintray.com/openhab/mvn/download_file?file_path=org%2Fopenhab%2Fdistro%2Fopenhab%2F2.2.0%2Fopenhab-2.2.0.zip) or the latest Snapshot Version [![Download](https://api.bintray.com/packages/openhab/mvn/openhab-distro/images/download.svg?version=2.3.0)](https://openhab.ci.cloudbees.com/job/openHAB-Distribution/lastSuccessfulBuild/artifact/distributions/openhab/target/openhab-2.3.0-SNAPSHOT.zip) of openHAB.
```

Comment:
I would remove the cloudbees links completely and point to `https://www.openhab.org/download/` instead.
That is what i wanted to do anyway for all installation articles in the future.
See #711 for reference.

It simply doesn't make sense for me, to fix those links after every release on every article.

Reply:
I was thinking about that solution as well. I'll remove them!

Reply:
I have also directly opened openhab/website#78 too get this a bit smoother in the future, if possible.

Reply:
Excellent! That would indeed make it a bit easier and less confusing. :smile: